frontend/src/components/namespace/CreateNamespaceButton.tsx

```diff
@@ -143,15 +144,15 @@ export default function CreateNamespaceButton() {
         </DialogContent>
         <DialogActions>
           <Button
-            aria-label="Cancel"
+            data-testid="create-namespace-dialog-cancel-button"
             onClick={() => {
               setNamespaceDialogOpen(false);
             }}
           >
             {t('translation|Cancel')}
           </Button>
           <Button
-            aria-label="Create"
+            data-testid="create-namespace-dialog-create-button"
```

Comment:
Why is this aria-label deleted?

Reply:
there's already a descriptive (and translated) label for these buttons